tutorials/hipe4ml_tutorial_multiclass.ipynb

```diff
@@ -0,0 +1,541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "iveMg4lZuOIK"
+   },
+   "source": [
+    "# **Multiclass classification with hipe4ml**\n",
+    "\n",
+    "The goal of this tutorial is to provide an example of multiclass classification with machine learing techniques using the package [hipe4ml](https://hipe4ml.github.io/).\n",
+    "\n",
+    "This tutorial is based on the particle identification using the information of the particle energy loss $\\mathrm{d}E/\\mathrm{d}x$ in the ITS and TPC detectors of the ALICE experiment in Run 1. The data were collected by ALICE in 2010 and correspond to 240 million minimum-bias pp collisions at $\\sqrt{s}$ = 7 TeV. In this data sample 200000 electrons, pions, kaons, protons, and deuterons were selected with different techniques. In particular, pions were selected via $\\mathrm{K_S^0}\\rightarrow\\mathrm{\\pi^+\\pi^-}$ decays, protons via $\\Lambda\\rightarrow\\mathrm{p\\pi^-}$ decays, kaons via the kink topology of $\\mathrm{K^+}\\rightarrow\\mathrm{\\pi^+\\nu_\\mu}$ decays, electrons via $\\gamma\\rightarrow\\mathrm{e^+e^-}$ conversions, and deuterons using the information from the time-of-flight detector.\n",
+    "\n",
+    "All the data samples can be found at:\n",
+    "\n",
+    "- pions: https://github.com/hipe4ml/hipe4ml_tests/blob/master/PID7TeV/pi_fromV0_data_filtered.parquet.gzip\n",
+    "- kaons: https://github.com/hipe4ml/hipe4ml_tests/blob/master/PID7TeV/kaons_fromkinks_data_filtered.parquet.gzip \n",
+    "- protons: https://github.com/hipe4ml/hipe4ml_tests/blob/master/PID7TeV/protons_fromL_data_filtered.parquet.gzip\n",
+    "- electrons: https://github.com/hipe4ml/hipe4ml_tests/blob/master/PID7TeV/electrons_fromconversions_data_filtered.parquet.gzip\n",
+    "- deuterons: https://github.com/hipe4ml/hipe4ml_tests/blob/master/PID7TeV/deuterons_fromTOFTPC_data_filtered.parquet.gzip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QM-yzbqwNqdX"
+   },
+   "source": [
+    "### **Prerequisites**\n",
+    "\n",
+    "The following packages are needed to follow this tutorial and in general to use _hipe4ml_:\n",
+    "\n",
+    "- [_uproot_](https://github.com/scikit-hep/uproot) is a reader and a writer of the ROOT file data format and is used to cast blocks of data from the ROOT file as Numpy arrays.\n",
+    "\n",
+    "- _xxhash_ and _lz4_ are used to decompress the files after download.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "gpdg8u8jBrmH"
+   },
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install hipe4ml\n",
+    "!pip install uproot\n",
+    "!pip install xxhash\n",
+    "!pip install lz4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **Restart Runtime(temporary)** \n",
```

Comment:
Is this still needed?

Reply:
I tested it and it is still needed.

Reply:
Thanks for testing it, indeed it is still needed